### DISCUSSION TITLE
no attribute 'MAX_RUN'

### DISCUSSION THREAD

**Discussion by awells96** [SciTools/iris#5179] [Q&A] [answered]
Hi,

I'm trying to calculate the longest run of precipitation exceeding 1 mm/day.  I found this example in the docs: https://scitools-iris.readthedocs.io/en/stable/generated/api/iris/analysis.html#iris.analysis.MAX_RUN 

For example:

The longest run of days with precipitation exceeding 10 (in cube data units) at each grid location could be calculated with:

result = precip_cube.collapsed('time', iris.analysis.MAX_RUN,
                               function=lambda values: values > 10)

However, when I try this I get the following error 
AttributeError: module 'iris.analysis' has no attribute 'MAX_RUN'

I don't get this error with any of the other aggregator instances that I've tried. 

Thanks in advance for your help!

**Comment by trexfeathers** [comment, marked as answer]
Hi @awells96, thanks for getting in touch. `iris.analysis.MAX_RUN` was introduced fairly recently - Iris `v3.3` - so there is a chance that your environment is simply using an older version of Iris.

What version of Iris do you get when you run the following in the command line?

```shell
python -c "import iris; print(iris.__version__)"
```

**Reply by awells96** [reply]
Hi, that was the problem! My environment was using 3.2.1. It's been updated now and it working as expected - thanks so much for your help!
